lib/howl/command.moon

```diff
@@ -204,8 +204,8 @@ interact.register
       .items = get_command_items!
       .headers = { 'Command', 'Key binding', 'Description' }
       .columns = {
-        { style: 'string' }
-        { style: 'keyword' }
+        { style: 'string', min_width: 30 }
```

Comment:
That means one very long command could push the entire column to be pretty wide - maybe not a big deal. I'll try it out.